examples/chat.cpp

```diff
@@ -70,34 +69,33 @@ class ChatServer: public Hub
 	}
 };
 
-class ChatClient: public Hub
+class ChatClient : public Hub
 {
 public:
-	ChatClient(string remoteTarget, const string& nick) :
-		inputStream(0), nick(nick)
+	ChatClient(string remoteTarget, const string& nick) : inputStream(0), nick(nick)
```

Comment:
Why changing this? I think that depending on the specific class one or the other option might be better.